specs-go/config.go

```diff
@@ -529,7 +529,8 @@ type LinuxSeccompArg struct {
 
 // LinuxSyscall is used to match a syscall in Seccomp
 type LinuxSyscall struct {
-	Name   string             `json:"name"`
-	Action LinuxSeccompAction `json:"action"`
-	Args   []LinuxSeccompArg  `json:"args,omitempty"`
+	Names   []string           `json:"names,omitempty"`
```

Comment:
You probably no longer need an `omitempty` here now that `Name` has been removed.